powerauth-java-cmd-lib/src/main/java/io/getlime/security/powerauth/lib/cmd/steps/v3/RemoveTokenStep.java

```diff
@@ -160,41 +162,42 @@ public JSONObject execute(StepLogger stepLogger, Map<String, Object> context) th
                 TypeReference<ObjectResponse<TokenRemoveResponse>> typeReference = new TypeReference<ObjectResponse<TokenRemoveResponse>>() {};
                 ObjectResponse<TokenRemoveResponse> responseWrapper = RestClientConfiguration
                         .defaultMapper()
-                        .readValue(response.getRawBody(), typeReference);
+                        .readValue(response.getBody(), typeReference);
 
                 if (stepLogger != null) {
-                    stepLogger.writeServerCallOK(responseWrapper, HttpUtil.flattenHttpHeaders(response.getHeaders()));
+                    stepLogger.writeServerCallOK("token-remove-response-received", responseWrapper, HttpUtil.flattenHttpHeaders(response.getHeaders()));
                 }
 
                 if (stepLogger != null) {
                     stepLogger.writeItem(
+                            "token-remove-token-removed",
                             "Token successfully removed",
                             "Token was successfully removed",
                             "OK",
                             responseWrapper.getResponseObject().getTokenId()
 
                     );
-                    stepLogger.writeDoneOK();
+                    stepLogger.writeDoneOK("token-remove-success");
                 }
 
                 return model.getResultStatusObject();
             } else {
                 if (stepLogger != null) {
-                    stepLogger.writeServerCallError(response.getStatus(), response.getBody(), HttpUtil.flattenHttpHeaders(response.getHeaders()));
-                    stepLogger.writeDoneFailed();
+                    stepLogger.writeServerCallError("token-remove-error-server-call", response.getStatus(), response.getBody(), HttpUtil.flattenHttpHeaders(response.getHeaders()));
+                    stepLogger.writeDoneFailed("token-remove-finished");
                 }
                 return null;
             }
         } catch (UnirestException exception) {
             if (stepLogger != null) {
-                stepLogger.writeServerCallConnectionError(exception);
-                stepLogger.writeDoneFailed();
+                stepLogger.writeServerCallConnectionError("token-remove-error-connection", exception);
+                stepLogger.writeDoneFailed("token-remove-finished");
             }
             return null;
         } catch (Exception exception) {
             if (stepLogger != null) {
-                stepLogger.writeError(exception);
-                stepLogger.writeDoneFailed();
+                stepLogger.writeError("token-remove-error-generic", exception);
+                stepLogger.writeDoneFailed("token-remove-finished");
```

Comment:
Should be `token-remove-failed`, for all three `writeDoneFailed()` calls in this file.